src/components/addSourceWizard/hardcodedComponents/azure/costManagement.js

```diff
@@ -124,19 +125,78 @@ const InternalReaderRoleDescription = () => {
 
   let scope = application?.extra?.scope || `/subscriptions/${application?.extra?.subscription_id}`;
 
-  return (
-    <TextContent>
-      <Text component={TextVariants.p}>
-        {intl.formatMessage({
-          id: 'cost.azure.createReaderRole',
-          defaultMessage: 'Run the following command in Cloud Shell to create a Cost Management Reader role:',
-        })}
-      </Text>
-      <ClipboardCopy>
-        {`az role assignment create --assignee "${authentication?.username}" --role "Cost Management Reader" --scope "${scope}"`}
-      </ClipboardCopy>
-    </TextContent>
-  );
+  if (scope.includes('billingAccounts')) {
+    if (scope.includes('enrollmentAccounts')) {
```

Comment:
Just a question about this if, does this change just the titles and texts? If so, we might want to change the texts directly instead of copying same code.

Reply:
Sorry, I don't quite understand the comment here.  The if condition changes the text and I wanted to make sure each string was translatable. Open to further suggestions though. I'm definitely not a true FE dev.